pkg/asset/internal/templates.go

```diff
@@ -282,9 +282,29 @@ metadata:
   labels:
     k8s-app: kube-controller-manager
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
With `replicas=2` and one node, one replica pod will fail to be scheduled due to pod anti affinity. The error log might confuse the user so I changed the replica number to 1. We could scale this to 2 and etcd pods to 3 when we have multiple master nodes available.